src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/enums/PrisonerSupportedAlertCodeType.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.dto.enums
+
+@Suppress("unused")
+enum class PrisonerSupportedAlertCodeType(val type: String, val description: String) {
+  C1("Child Communication Measures", "L1 Restriction No contact with any child"),
```

Comment:
We don't want to store the description as this is just duplicating of what the code provides... If it is is a hard coded description the code has all that we need. I think we need either a join table or a JPA abstraction class